src/inversion_scripts/jacobian.py

```diff
@@ -120,7 +121,8 @@ def apply_operator(operator, params):
 
     # Map GEOS-Chem to TROPOMI observation space
     # Also return Jacobian matrix if build_jacobian=True
-    for filename in sat_files:
+    def process(filename):
+    # for filename in sat_files:
```

Comment:
remove comment here

Reply:
Now removed in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/c53f4d519860a678a5dec31f403770c497cfab43